core/merginapi.cpp

```diff
@@ -29,8 +29,10 @@
 #include <geodiff.h>
 
 const QString MerginApi::sMetadataFile = QStringLiteral( "/.mergin/mergin.json" );
+const QString MerginApi::sMerginConfigFile = QStringLiteral( "/.mergin-config.json" );
 const QString MerginApi::sDefaultApiRoot = QStringLiteral( "https://public.cloudmergin.com/" );
 const QSet<QString> MerginApi::sIgnoreExtensions = QSet<QString>() << "gpkg-shm" << "gpkg-wal" << "qgs~" << "qgz~" << "pyc" << "swap";
+const QSet<QString> MerginApi::sIgnoreImageExtensions = QSet<QString>() << "jpg" << "JPEG" << "png";
```

Comment:
From what I understood, Martin was talking about having extensions in the config file too, is that right @wonder-sk? So that in plugin you can ignore any particular file extension.

Reply:
ATM its hardcoded according the discussion we had - most common images extensions are supported (not case sensitive), but the list can be easily extended.